### PR TITLE
Feature/slider

### PR DESCRIPTION
**Pull request by jsp-up**

### CODE REVIEW

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-slider/kal-slider.component.ts [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+})
+export class KalSliderComponent extends FormControlAccessComponent<number> implements OnInit, OnDestroy {
+
+  /**
+   * Emits when the value of the slider changes.
+   */
+  @Output() readonly valueChange: EventEmitter<number> = new EventEmitter<number>();
+
+  control = new FormControl(0);
+  private sliderValue = 0;
```
Comment (MichaelGrosio) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-slider/kal-slider.component.ts:37]: spaces in component

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-slider/kal-slider.component.ts [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  providers: buildProviders(KalSliderComponent)
+})
+export class KalSliderComponent extends FormControlAccessComponent<number> implements OnInit, OnDestroy {
+
+  /**
+   * Emits when the value of the slider changes.
+   */
+  @Output() readonly valueChange: EventEmitter<number> = new EventEmitter<number>();
+
+  control = new FormControl(0);
```
Comment (MichaelGrosio) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-slider/kal-slider.component.ts:35]: Comments in component

**Review by Bellico — DISMISSED** [review]

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-slider/kal-slider.component.ts [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    this.displayThumbLabel = coerceBooleanProperty(value);
+    this.cdr.markForCheck();
+  }
+
+  /**
+   * How often to show ticks. Relative to the step so that a tick always appears on a step.
+   * Ex: Tick interval of 4 with a step of 3 will draw a tick every 4 steps (every 12 values).
+   */
+  @Input()
+  get tickInterval() {
```
Comment (MichaelGrosio) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-slider/kal-slider.component.ts]: Type

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-slider/kal-slider.component.ts [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+  }
+  set limit(value: number) {
+    this.limitSliderValue = coerceNumberProperty(value);
+
+    if (this.limit !== 0 && this.limit < this.value) {
+      this.writeValue(this.limit);
+    }
+  }
+
+  @Input()
```
Comment (MichaelGrosio) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-slider/kal-slider.component.ts:75]: Comments in component

**Review by ydomenjoud — APPROVED** [review]

**Pull request merged**